app/code/core/Mage/Cms/Model/Resource/Block.php

```diff
@@ -63,8 +59,9 @@ protected function _beforeDelete(Mage_Core_Model_Abstract $object)
     /**
      * Perform operations before object save
      *
-     * @param Mage_Cms_Model_Block $object
+     * @param Mage_Core_Model_Abstract $object
```

Comment:
Mage_Cms_Model_Block is correct

Reply:
Yes and no.

`Mage_Cms_Model_Block`  is expected, but signature says ... `protected function _beforeSave(Mage_Core_Model_Abstract $object)`

I'll update to `Mage_Core_Model_AbstractMage_Cms_Model_Block |`

Reply:
ok, this will work.
I saw that you did add different type to doc comment then typehint in other places. even in this patch. see:

``` 
    * @param Mage_Cms_Model_Page $object
     * @inheritDoc
     */
    protected function _beforeSave(Mage_Core_Model_Abstract $object)

```